### PR TITLE
askama_axum: prepare version 0.1.0

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: askama_axum/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "askama_axum"
-version = "0.10.0"
+version = "0.1.0"
```
Comment (Kijewski) [inline askama_axum/Cargo.toml:3]: Um, I guess you meant to hit the "1" key one more often

**Review by Kijewski — CHANGES_REQUESTED** [review]

File: askama_axum/Cargo.toml (hunk 1 above)
Comment (djc) [reply askama_axum/Cargo.toml:3]: I actually didn't... This is a new package that hasn't been published before, so we can just start from 0.1.0.

**Comment by djc** [comment]
The failure here is unrelated to the changes; it will be fixed by #585.

**Comment by djc** [comment]
Ping? Would be nice to get this out.

**Review by Kijewski — APPROVED** [review]
Looks good to me!

**Pull request merged**